juju/juju
# New Migration-Master API Method to Retrieve Data for Cross-Model Pre-Checks

## Description of change

This patch adds a new method to the migration-master API server. 

It returns details that the corresponding worker needs in order to do migration pre-checks on external controllers consuming offers in the migrating model.

That check logic will be added in a subsequent patch.

## QA steps

Unit tests confirm behaviour. Once the external pre-check logic is in place, we will be able to verify via QA steps and integration tests.

## Documentation changes

None.

## Bug reference

N/A
